extension/copy-gist.js

```diff
@@ -0,0 +1,25 @@
+/* globals utils */
+
+'use strict';
+
+window.addGistCopyButton = () => {
+	// Button already added (partial page nav), or non-text file
+	if ($('.copy-btn').length > 0) {
+		return;
+	}
+	const $gistsSibling = $('.file-actions > .btn.btn-sm');
+
+	for (let i = 0; i < $gistsSibling.length; i++) {
+		const sourceCodeClass = $(`#${$gistsSibling[i].offsetParent.id}>div`).eq(1)[0].className;
```

Comment:
This is a bit inefficient and convoluted. Would be better to find the common container for both the buttons and the source code and then find the source code from there.